analyzers/SonarAnalyzer.sln

```diff
@@ -24,8 +24,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SonarAnalyzer.CSharp", "src
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SonarAnalyzer.VisualBasic", "src\SonarAnalyzer.VisualBasic\SonarAnalyzer.VisualBasic.csproj", "{7CFA8FA5-8842-4E89-BB90-39D5C0F20BA8}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ReviewDiffs", "its\ReviewDiffs\ReviewDiffs.csproj", "{7E3ADCD6-01D7-44E2-B12F-C485C35C8A9C}"
```

Comment:
The tool is mentioned in [docs/contributing-analyzer.md](https://github.com/SonarSource/sonar-dotnet/blob/8b92d62b37efb782ec13a92a0903c78f5e8d097d/docs/contributing-analyzer.md). Remove the paragraph about it.